code_fellows/deque.py

```diff
@@ -0,0 +1,107 @@
+# https://codefellows.github.io/sea-python-401d6/assignments/deque.html
+
+
+class Deque:
+    def __init__(self, value=None):
+        self.head = None
+        self.tail = None
+        self._length = 0
+        self.is_empty = True
```

Comment:
I don't think there is benefit of is_empty over just using the length.  Especially since really you shouldn't check at all for most these case and instead just _try_ the thing and handle the failure case.

Reply:
agreed. The next commit will remove all of the .is_empty.